hw/xbox/nv2a/gl/Makefile.objs

```diff
@@ -2,3 +2,4 @@ obj-y += gloffscreen_common.o
 obj-$(CONFIG_WIN32) += gloffscreen_wgl.o
 obj-$(CONFIG_DARWIN) += gloffscreen_cgl.o
 obj-$(CONFIG_LINUX) += gloffscreen_glx.o
+obj-$(CONFIG_BSD) += gloffscreen_glx.o
```

Comment:
MacOS is both, `DARWIN` and `BSD`, so you'd probably need something like:
```
ifeq ($(CONFIG_DARWIN),y)
obj-$(CONFIG_DARWIN) += gloffscreen_cgl.o 
else
obj-$(CONFIG_BSD) += gloffscreen_glx.o 
endif
```

or (less QEMU-ish, because they never seem to check with `ifneq` for those `CONFIG_` flags):

```
ifneq ($(CONFIG_DARWIN),y)
obj-$(CONFIG_BSD) += gloffscreen_glx.o 
endif
```

I'm not sure if there's a more elegant way

Reply:
This line its not correct ?

Reply:
I changed to right code style